.gitignore

```diff
@@ -15,6 +15,7 @@ requests.egg-info/
 *.swp
 *.egg
 env/
+.idea/
```

Comment:
No, I'm recommending you remove this line and use a [Global gitignore file](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).